[2024-06-24T06:16:30.842+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T06:16:30.934+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final_1.check_previous_run_status manual__2024-06-24T06:09:24.609873+00:00 [queued]>
[2024-06-24T06:16:30.959+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final_1.check_previous_run_status manual__2024-06-24T06:09:24.609873+00:00 [queued]>
[2024-06-24T06:16:30.960+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 6
[2024-06-24T06:16:31.007+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-24 06:09:24.609873+00:00
[2024-06-24T06:16:31.039+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_final_1', 'check_previous_run_status', 'manual__2024-06-24T06:09:24.609873+00:00', '--job-id', '3309', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting.py', '--cfg-path', '/tmp/tmpi4on1a8c']
[2024-06-24T06:16:31.028+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=92) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T06:16:31.041+0000] {standard_task_runner.py:91} INFO - Job 3309: Subtask check_previous_run_status
[2024-06-24T06:16:31.042+0000] {standard_task_runner.py:63} INFO - Started process 103 to run task
[2024-06-24T06:16:31.265+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_final_1.check_previous_run_status manual__2024-06-24T06:09:24.609873+00:00 [running]> on host bf895605831d
[2024-06-24T06:16:31.927+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_final_1' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T06:09:24.609873+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-24T06:09:24.609873+00:00'
[2024-06-24T06:16:31.928+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T06:16:31.982+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-24T06:16:32.025+0000] {python.py:237} INFO - Done. Returned value was: start_run
[2024-06-24T06:16:32.026+0000] {branch.py:36} INFO - Branch into start_run
[2024-06-24T06:16:32.027+0000] {skipmixin.py:178} INFO - Following branch start_run
[2024-06-24T06:16:32.116+0000] {skipmixin.py:246} INFO - Skipping tasks [('skip_run', -1)]
[2024-06-24T06:16:32.184+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T06:16:32.244+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_final_1, task_id=check_previous_run_status, run_id=manual__2024-06-24T06:09:24.609873+00:00, execution_date=20240624T060924, start_date=20240624T061630, end_date=20240624T061632
[2024-06-24T06:16:32.302+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T06:16:32.331+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
